src/desktop/apps/authentication/components/meta.tsx

```diff
@@ -19,6 +20,7 @@ export const AuthenticationMeta: React.SFC<MetaProps> = props => {
       <meta property="description" content={description} />
       <meta property="og:description" content={description} />
       <meta property="twitter:description" content={description} />
+      {canonical && <link rel="canonical" href={canonical} />}
       {/* Don't index or follow links on this page */}
       <meta name="robots" content="none" />
```

Comment:
(non-blocking) @jonallured do you how the `robots` meta tag interacts with canonicals? It seems like adding self-referencing canonical tag wouldn't make a difference if we're explicitly telling search engines not to index these routes.

Reply:
More chatter here: https://artsy.slack.com/archives/C01ADJNCS5D/p1612460986337500